ignite/engine/engine.py

```diff
@@ -547,6 +548,9 @@ def terminate(self) -> None:
         - :attr:`~ignite.engine.events.Events.TERMINATE`
         - :attr:`~ignite.engine.events.Events.COMPLETED`
 
+        Args:
+            skip_completed: if True, the event `~ignite.engine.events.Events.COMPLETED` is not fired after
+            `~ignite.engine.events.Events.TERMINATE`. Default is False.
```

Comment:
There is a warning when building docs here: https://app.netlify.com/sites/pytorch-ignite-preview/deploys/674e251e13e2fa000828b048

Links to the events should be actually:
```
:attr:`~ignite.engine.events.Events.COMPLETED`
:attr:`~ignite.engine.events.Events.TERMINATE`
```